corehq/apps/es/REINDEX_PROCESS.md

```diff
@@ -0,0 +1,366 @@
+            ```
+
+            Ensure that `search_failures` and `indexing_failures` are empty lists.
+
+        2. Then check doc counts between primary and secondary indices using:
+
+            ```
+            cchq env django-manage elastic_sync_multiplexed display_doc_counts <index_cname>
+            ```
+
+            This command will display the document counts for both the primary and secondary indices for a given index. If the doc count matches between the two and there are no errors in the reindex logs, then reindexing is complete for that index.
+
+            Please note that for high frequecy indices like case_search, cases and forms the counts may not match perfectly. In such cases, ensure the difference in counts is small (within hundred) and there are no errors in reindex logs.
+
+    5.  After the index has been reindexed, we need to cleanup tombstones and setting the correct replica count for the newly created index.
+        ```
+        ./manage.py elastic_sync_multiplexed cleanup <index_cname>
+        ```
+
+        ```
+        ./manage.py elastic_sync_multiplexed set_replicas <index_cname>
+        ```
+    6. At this step we should have a secondary index for `<index_cname>` which has same data as of primary index. On index `<index_cname>` writes will continue to happen to both primary and secondary indices simultaneously. Reads would still happening from the primary index.
+
+    7. To switch the reads from primary index to the new secondary index, we need to swap the index.
+    To swap the index we need to follow the following steps:
+
+        1. Stop the pillows
+
+            ```
+            cchq <env> service pillowtop stop
+            ```
+        2. Copy the checkpoint ids for the pillows that depend on the index you are about to swap.
+
+            ```
+            cchq <env> django-manage  elastic_sync_multiplexed copy_checkpoints <index_cname>
+            ```
+        3. We will now swap the index in settings file to  make the primary index as secondary and vice versa.
+        Update `environments/<env>/public.yml`, set
+            ```
+            ES_<index_cname>_INDEX_SWAPPED = True
+            ```
+
+        4. From control machine, run update-config and restart commcare services so that new settings are applied. This will also restart the pillows that were stopped in previous step.
+            ```
+            cchq <env> update-config
+            cchq <env> service commcare restart
+            ```
+            After this step, the indexes will be swapped and reads will now happen from the newly created secondary indices.
+
+        5. Look around in CommcareHQ, test out things dealing with the index that was swapped
+
+        6. It is recommended to keep the indices in this state for at least 1 working day. This will provide a safe window to fall back if needed.
```

Comment:
```suggestion
        6. It is recommended to keep the index in this state for at least 1 working day. This will provide a safe window to fall back if needed.
```